docs/support.md

```diff
@@ -0,0 +1,13 @@
+# Support
+If you are having any issues in installing, configuring, or otherwise issues with KMK, please reach out to us and
+our community here. We do have a bridge to allow communication between the platforms as well, though we recommend
+using Matrix if possible.
+
+# Matrix
+* [Software Support](https://matrix.to/#/#kmk-support:kmkfw.io)
+* [Hardware Support](https://matrix.to/#/#kmk-hardware:kmkfw.io)
+* [General discussion](https://matrix.to/#/#kmk-general:kmkfw.io)
+
+
+# Discord
+* [General support](https://discord.gg/NDUau62)
```

Comment:
These invite links expire after a day or two if I recall correctly? Which is why there's bot infrastructure in other projects to support this mess?

If the link still works going forward, cool.

Reply:
It's a perma invite. It's good, made sure to check that.